pkg/cmd/pr/create/create.go

```diff
@@ -561,11 +571,12 @@ func submitPR(opts CreateOptions, ctx CreateContext, state shared.IssueMetadataS
 	client := ctx.Client
 
 	params := map[string]interface{}{
-		"title":       state.Title,
-		"body":        state.Body,
-		"draft":       state.Draft,
-		"baseRefName": ctx.BaseBranch,
-		"headRefName": ctx.HeadBranchLabel,
+		"title":               state.Title,
+		"body":                state.Body,
+		"draft":               state.Draft,
+		"baseRefName":         ctx.BaseBranch,
+		"headRefName":         ctx.HeadBranchLabel,
+		"maintainerCanModify": !opts.MaintainerCantModify,
```

Comment:
`!opts.MaintainerCantModify` reads a bit convoluted. Couldn't we have `MaintainerCanModify` struct field and assign to it the reverse value of the `--no-maintainer-edit` flag?

Reply:
my par was so low tho